### PR TITLE
Merge in the September release Branch to Dev

### PR DESCRIPTION
**Pull request by wainaina**
Merges in the September release Branch to Dev

### CODE REVIEW

**Review by melissaahn — APPROVED** [review]

**Review by bktowett — APPROVED** [review]

File: msal/build.gradle [hunk 1]
```diff
@@ -181,7 +181,7 @@ task sourcesJar(type: Jar) {
 
 // In dev, we want to keep the dependencies(common4j, broker4j, common) to 0.0.+ to be able to be consumed by daily dev pipeline.
 // In release/*, we change these to specific versions being consumed.
-String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "0.0.+"
+String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "7.0.1"
```
Comment (iamgusain) [inline msal/build.gradle]: Please revert this change. 
See comments in L182-183

File: changelog [hunk 2]
```diff
@@ -1,6 +1,9 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 V.Next
 ----------
+
+Version 4.0.2
+----------
```
Comment (iamgusain) [inline changelog:6]: Can you add the changelog entry for updating common version.
- [PATCH] Update common @7.0.0

File: msal/build.gradle (hunk 1 above)
Comment (wainaina) [reply msal/build.gradle]: Change effected.
